dht.go

```diff
@@ -504,9 +561,6 @@ func (dht *IpfsDHT) peerStoppedDHT(ctx context.Context, p peer.ID) {
 	// A peer that does not support the DHT protocol is dead for us.
 	// There's no point in talking to anymore till it starts supporting the DHT protocol again.
 	dht.routingTable.RemovePeer(p)
-
-	// since we lost a peer from the RT, we should do this here
-	dht.fixRTIfNeeded()
```

Comment:
Should we not do this? The idea was to fix our routing table if we drop below some number of peers.

Reply:
I guess the periodic bootstrap will help.

Reply:
@Stebalien Should have added a comment here. The `dht.routingTable.RemovePeer` call will result into a call to the Peer Removed callback which in turns calls `fixRTIfNeeded `.